README.md

```diff
@@ -130,6 +130,7 @@
 - [Frequently Asked Questions (FAQ)](FAQ.md)
 - [TiDB Best Practices](https://pingcap.github.io/blog/2017/07/24/tidbbestpractice/)
 + [Releases](releases/rn.md)
+  - [2.1 RC1](releases/2.1rc1.md)
```

Comment:
Pls keep the file name consistent with that in docs-cn, so it can jump to the Chinese version.